ksql-cli/src/test/java/io/confluent/ksql/cli/CliTest.java

```diff
@@ -536,9 +537,9 @@ public void testSelectProject() {
 
     final PhysicalSchema resultSchema = PhysicalSchema.from(
         LogicalSchema.builder()
-            .valueColumn("ITEMID", SqlTypes.STRING)
-            .valueColumn("ORDERUNITS", SqlTypes.DOUBLE)
-            .valueColumn("PRICEARRAY", SqlTypes.array(SqlTypes.DOUBLE))
+            .valueColumn(ColumnName.of("ITEMID"), SqlTypes.STRING)
```

Comment:
I don't think we gain anything by changing the parameter here to a `ColumnName` from a `String`.... the function is adding a column, so yes, the supplied parameter is a name.  Internally, sure this should be a `ColumnName`.

Thoughts?

Reply:
If this was just used in testing code I would've agreed with you, the concern is that it's also used to "copy" schemas around, which I wanted to keep type safe.